README.md

```diff
@@ -1,159 +1,48 @@
-# ScopeLift Foundry Template
+# Flexible Voting Clients 
 
-An opinionated template for [Foundry](https://github.com/foundry-rs/foundry) projects.
+[Flexible Voting](https://github.com/ScopeLift/flexible-voting) is an extension to the widely used OpenZeppelin DAO Governor that enables novel voting patterns for delegates. Clients are contracts that leverage Flexible Voting to allow they these novel voting patterns, such as:
 
-_**Please read the full README before using this template.**_
 
-- [Usage](#usage)
-- [Overview](#overview)
-  - [`foundry.toml`](#foundrytoml)
-  - [CI](#ci)
-  - [Test Structure](#test-structure)
-- [Configuration](#configuration)
-  - [Coverage](#coverage)
-  - [Slither](#slither)
-  - [GitHub Code Scanning](#github-code-scanning)
+  - Voting with tokens while earning yield in DeFi
+  - Voting with tokens bridged to L2
+  - Shielded voting (i.e. secret/private voting)
+  - Cheaper subsidized signature based voting
+  - Better voting options with tokens held by custodians
 
-## Usage
+Flexible Voting Clients contains existing clients and the tools to build new clients.
```

Comment:
I don't understand what this sentence means

Reply:
How about "The Flexible Voting Clients repo contains existing clients and the necessary tools to build new clients.
"? I am basically trying to say it contains client implementations and helper contracts for people to build their own clients.

Reply:
That sgtm, I didn't realize the first part of the sentence was referring to the repo, makes sense this way. Feel free to resolve once the change is made

Reply:
https://github.com/ScopeLift/flexible-voting-clients/commit/a1360f71c59b43876407ff588e5e8676eb04f7fc